docs/installer-stand-alone.md

```diff
@@ -0,0 +1,136 @@
+Assisted Installer Stand-Alone
+==============================
+
+This document describes the process of running the Assisted Installer in
+stand-alone mode via `podman`.
+
+# Pre-Requisites
+
+## OpenShift User Pull Secret
+
+You will need a valid OpenShift user pull secret. Copy or download the pull
+secret from https://cloud.redhat.com/openshift/install/pull-secret
+
+## Download CoreOS Installer
+
+This is used by the assisted-service to generate the discovery ISO. You can grab
+it from a released container image by:
+
+```
+podman run --privileged --pull=always -it --rm \
+  -v .:/data --entrypoint /bin/cp \
+  quay.io/coreos/coreos-installer:v0.7.0 /usr/sbin/coreos-installer /data/coreos-installer
+```
+
+## Download RHCOS Live ISO
+
+The RHCOS Live ISO is used as the base when building discovery ISOs. These
+images can be found at https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos
+Later on, in this document, you will see it referred to as `livecd.iso`.
+
+**NOTE** 
+The RHCOS images might not change with every release of OpenShift Container Platform.
+You must download images with the highest version that is less than or equal to the
+OpenShift Container Platform version that you install. Use the image versions that
+match your OpenShift Container Platform version if they are available.
+
+# Running the Assisted Installer using Podman
+
+## Environment
+
+The first thing you will need to do is grab the
+[`onprem-environment`](https://raw.githubusercontent.com/openshift/assisted-service/master/onprem-environment)
+file. Once you have this file, and have modified it based on your needs -- if
+you need the Assisted Installer UI to run on a different port to avoid conflict
+with `httpd` as an example -- you should source it.
+
+```
+source onprem-environment
+```
+
+**NOTE**
+The remainder of this document relies on the values stored in
+`onprem-environment` being set in the shell.
+
+## NGINX Configuration
+
+Once you have your environment setup, you will need to grab the
+[`nginx.conf`](https://raw.githubusercontent.com/openshift/assisted-service/master/deploy/ui/nginx.conf)
+used to configure the Assisted Installer's UI. With the `nginx.conf` file in
+your current working directory, you will want to update it to reflect any
+changes you made to `onprem-environment` previously.
+
+```
+# Configure the port used to access the Assisted Installer UI
+sed -i "s|listen.*;|listen $UI_PORT;|" nginx.conf
+
+# Configure the port used by the Assisted Installer to acces the inventory
+sed -i "s|proxy_pass.*;|proxy_pass $SERVICE_BASE_URL;|" nginx.conf
+```
+
+## Create the Assisted Installer Pod
+
+Once you have made any adjustments to ports as necessary, you can create the
+assisted-installer pod.
+
+```
+podman pod create --name assisted-installer -p ${DB_PORT},${INVENTORY_PORT},${UI_PORT}
```

Comment:
This command will failed since there was not defined INVENTORY_PORT and UI_PORT.
[root@ocp-edge33 ~]# podman pod create --name assisted-installer -p ${DB_PORT},${INVENTORY_PORT},${UI_PORT}
Error: unable to create pod: No port specified: <empty>
[root@ocp-edge33 ~]# grep DB_PORT onprem-environment 
DB_PORT=5432
[root@ocp-edge33 ~]# grep INVENTORY_PORT onprem-environment 
[root@ocp-edge33 ~]# grep UI_PORT onprem-environment 
[root@ocp-edge33 ~]# grep INVENTORY_PORT nginx.conf 
[root@ocp-edge33 ~]# grep UI_PORT nginx.conf

Reply:
Until this PR is merged. You would need to use this onprem-environment https://raw.githubusercontent.com/openshift/assisted-service/6be44e3776cb40bd185d041a2309161a2d1584dd/onprem-environment